src/libponyc/codegen/gencall.c

```diff
@@ -1019,6 +1019,7 @@ static LLVMTypeRef ffi_return_type(compile_t* c, reach_type_t* t,
 
   if(t->underlying == TK_TUPLETYPE)
   {
+    (void)intrinsic;
```

Comment:
i'm not sure how we want to deal with these things that aren't in an #ifdef that exist only for asserts.

this is my "get it to compile solution". any ideas for a better approach?